### DISCUSSION TITLE
How to test Tauon Satellite feature?

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#779] [Q&A]
How to test Tauon 7.3.0 Satellite new Experimental feature?
I don't see it in the settings.

**Comment by Taiko2k** [comment]
Should be under settings > accounts?

**Comment by Solarunit** [comment]
Found it!

![](https://i.imgur.com/m4fAvVt.png)

You need to know `ip:port`, what is the port btw?

import All playlists not available now?

**Reply by Taiko2k** [reply]
Just the ip. And yeah all playlists not available.
